src/Files.App/ViewModels/Settings/AppearanceViewModel.cs

```diff
@@ -42,7 +56,64 @@ public AppearanceViewModel(IUserSettingsService userSettingsService, IResourcesS
 
 			AppThemeResources = AppThemeResourceFactory.AppThemeResources;
 
+
+			// Background image fit options
+			ImageStretchTypes.Add(Stretch.None, "None".GetLocalizedResource());
+			ImageStretchTypes.Add(Stretch.Fill, "Fill".GetLocalizedResource());
+			ImageStretchTypes.Add(Stretch.Uniform, "Uniform".GetLocalizedResource());
+			ImageStretchTypes.Add(Stretch.UniformToFill, "UniformToFill".GetLocalizedResource());
+			SelectedImageStretchType = ImageStretchTypes[UserSettingsService.AppearanceSettingsService.AppThemeBackgroundImageFit];
+
+			// Background image allignment options
+
+			// VerticalAlignment
+			ImageVerticalAlignmentTypes.Add(VerticalAlignment.Top, "Top".GetLocalizedResource());
+			ImageVerticalAlignmentTypes.Add(VerticalAlignment.Center, "Center".GetLocalizedResource());
+			ImageVerticalAlignmentTypes.Add(VerticalAlignment.Bottom, "Bottom".GetLocalizedResource());
+			ImageVerticalAlignmentTypes.Add(VerticalAlignment.Stretch, "Stretch".GetLocalizedResource());
+			SelectedImageVerticalAlignmentType = ImageVerticalAlignmentTypes[UserSettingsService.AppearanceSettingsService.AppThemeBackgroundImageVerticalAlignment];
+
+			// HorizontalAlignment
+			ImageHorizontalAlignmentTypes.Add(HorizontalAlignment.Left, "Top".GetLocalizedResource());
+			ImageHorizontalAlignmentTypes.Add(HorizontalAlignment.Center, "Center".GetLocalizedResource());
+			ImageHorizontalAlignmentTypes.Add(HorizontalAlignment.Right, "Bottom".GetLocalizedResource());
+			ImageHorizontalAlignmentTypes.Add(HorizontalAlignment.Stretch, "Stretch".GetLocalizedResource());
+			SelectedImageHorizontalAlignmentType = ImageHorizontalAlignmentTypes[UserSettingsService.AppearanceSettingsService.AppThemeBackgroundImageHorizontalAlignment];
```

Comment:
* Shouldn't horizontal alignment be left, center, or right?
* Does `Stretch` make sense?

Reply:
I just went with all the available options, but we can remove stretch if it doesn't make a difference.

Reply:
Then we should remove stretch.